packages/odyssey-react/src/components/Table/TableSortButton.tsx

```diff
@@ -65,15 +69,21 @@ export const TableSortButton = withTheme(
     const omitProps = useOmit(rest);
 
     return (
-      <button {...omitProps} ref={ref} className={componentClass}>
+      <Box
+        as="button"
+        {...omitProps}
+        ref={ref}
+        className={componentClass}
+        lineHeight={false}
```

Comment:
curious why the `lineHeight={false}` prop is necessary here for Table components?

Reply:
[table sets a line height](https://github.com/okta/odyssey/blob/8262093e1307ef7772753d443e20ecbbb2d376a6/packages/odyssey-react/src/components/Table/Table.module.scss#L19) that it expects children to inherit,  if the sort button sets its own line height (via Box defaults) that adds additional vertical space and sets off VRT

Reply:
@andrewberg-okta perfect thank you